README


This archive contains the distribution Task-BeLike-SCHWIGON,
version 0.003:

  Modules SCHWIGON uses

This software is copyright (c) 2013 by Steffen Schwigon.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.




Changes
Revision history for Task-BeLike-SCHWIGON

0.003     2013-05-02
        - add Archive::Tar::Wrapper
        - add Task::PerlFormance
        - add DBD::Pg
        - add TAPPER deps
        - drop lots of unused stuff

0.002     2012-08-07
        - add: indirect, multidimensional, bareword::filehandles

0.001     2012-05-08
        - initial module collection roughly based on FLORA


lib/Task/BeLike/SCHWIGON.pm
use strict;
use warnings;

package Task::BeLike::SCHWIGON;
# git description: v0.002-3-g02a8d9b

BEGIN {
  $Task::BeLike::SCHWIGON::AUTHORITY = 'cpan:SCHWIGON';
}
{
  $Task::BeLike::SCHWIGON::VERSION = '0.003';
}
# ABSTRACT: Modules SCHWIGON uses


1;

__END__

=pod

=head1 NAME

Task::BeLike::SCHWIGON - Modules SCHWIGON uses

=head1 VERSION

version 0.003

=head1 TASK CONTENTS

=head2 Applications

=head3 L<App::Ack>

=head3 L<App::Nopaste>

=head3 L<App::Software::License>

=head3 L<App::cpanminus>

=head3 L<App::cpanoutdated>

=head3 L<CPAN::Mini>

=head3 L<CPAN::Mini::Devel>

=head3 L<CPAN::Uploader>

=head2 Optional Dependencies Of Other Modules

=head3 L<CPAN::Reporter>

=head3 L<PadWalker>

=head3 L<Task::SDK::Moose>

=head3 L<Term::ReadLine::Gnu>

=head3 L<Test::Reporter>

=head3 L<Test::Reporter::Transport::Socket>

=head2 Development Tools

=head3 L<Archive::Tar::Wrapper>

=head3 L<bareword::filehandles>

=head3 L<Devel::bt>

=head3 L<Devel::Cover>

=head3 L<Devel::NYTProf>

=head3 L<Dist::Zilla>

=head3 L<Dist::Zilla::PluginBundle::SCHWIGON>

=head3 L<Pod::Weaver::PluginBundle::SCHWIGON>

=head3 L<Dist::Zilla::PluginBundle::TAPPER>

=head3 L<Pod::Weaver::PluginBundle::TAPPER>

=head3 L<indirect>

=head3 L<Module::Install> 0.95

=head3 L<Module::Install::AuthorRequires>

=head3 L<Module::Install::AuthorTests>

=head3 L<Module::Install::CheckConflicts>

=head3 L<Module::Install::ExtraTests>

=head3 L<multidimensional>

=head3 L<Perl::Tidy>

=head3 L<Perl::Version> 1.010

=head3 L<Pod::Coverage::TrustPod>

=head3 L<Test::Aggregate>

=head3 L<Test::Pod>

=head3 L<Test::Pod::Coverage>

=head3 L<V>

=head2 Modules I use a lot

=head3 L<App::Cmd>

=head3 L<App::Rad>

=head3 L<Task::PerlFormance>

=head3 L<Bundle::CPANxxl>

=head3 L<Bundle::libnet>

=head3 L<Bundle::LWP>

=head3 L<Cache::Cache>

=head3 L<Cache::FileCache>

=head3 L<Data::DPath>

=head3 L<App::DPath>

=head3 L<Data::Visitor>

=head3 L<Data::YAML>

=head3 L<DateTime>

=head3 L<DateTime::Format::Builder>

=head3 L<DateTime::Format::Mail>

=head3 L<DateTime::Format::MySQL>

=head3 L<DateTime::Format::Natural>

=head3 L<DateTime::Format::Strptime>

=head3 L<DBD::mysql>

=head3 L<DBD::SQLite>

=head3 L<DBD::Pg>

=head3 L<DBI>

=head3 L<DBIx::Class>

=head3 L<DBIx::Class::Schema::Loader>

=head3 L<DBIx::Class::Schema::Versioned>

=head3 L<DBIx::Class::TimeStamp>

=head3 L<Devel::Backtrace>

=head3 L<Digest::SHA1>

=head3 L<Directory::Scratch>

=head3 L<File::ShareDir>

=head3 L<File::Slurp>

=head3 L<File::Type>

=head3 L<Hash::Merge>

=head3 L<Hash::Merge::Simple>

=head3 L<JSON>

=head3 L<JSON::Syck>

=head3 L<JSON::XS>

=head3 L<List::AllUtils>

=head3 L<List::MoreUtils>

=head3 L<Log::Log4perl>

=head3 L<Moose>

=head3 L<Moose::Autobox>

=head3 L<MooseX::Daemonize>

=head3 L<MooseX::Declare>

=head3 L<MooseX::LazyRequire>

=head3 L<MooseX::Log::Log4perl>

=head3 L<MooseX::Method::Signatures>

=head3 L<MooseX::Types::Common>

=head3 L<MooseX::Types::DateTime>

=head3 L<MooseX::Types::Email>

=head3 L<MooseX::Types::LoadableClass>

=head3 L<MooseX::Types::Path::Class>

=head3 L<MooseX::Types::URI>

=head3 L<Net::Daemon>

=head3 L<Net::Server>

=head3 L<Net::Server::PreForkSimple>

=head3 L<Net::SSH>

=head3 L<Net::SSH::Expect>

=head3 L<Net::TFTP>

=head3 L<Parse::RecDescent>

=head3 L<Regexp::Common>

=head3 L<SQL::Translator>

=head3 L<Sub::Exporter>

=head3 L<TAP::DOM>

=head3 L<TAP::Formatter::HTML>

=head3 L<TAP::Parser::Aggregator>

=head3 L<Template>

=head3 L<Term::ReadLine::Perl>

=head3 L<Test::Deep>

=head3 L<Test::Fixture::DBIC::Schema>

=head3 L<Test::MockModule>

=head3 L<Test::WWW::Mechanize::Catalyst>

=head3 L<TryCatch>

=head3 L<URI>

=head3 L<URI::Escape>

=head3 L<YAML>

=head3 L<YAML::Syck>

=head3 L<YAML::XS>

=head1 AUTHOR

Steffen Schwigon <ss5@renormalist.net>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2013 by Steffen Schwigon.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=cut


Makefile.PL

use strict;
use warnings;



use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  "ABSTRACT" => "Modules SCHWIGON uses",
  "AUTHOR" => "Steffen Schwigon <ss5\@renormalist.net>",
  "BUILD_REQUIRES" => {},
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => "6.30"
  },
  "DISTNAME" => "Task-BeLike-SCHWIGON",
  "EXE_FILES" => [],
  "LICENSE" => "perl",
  "NAME" => "Task::BeLike::SCHWIGON",
  "PREREQ_PM" => {
    "App::Ack" => 0,
    "App::Cmd" => 0,
    "App::DPath" => 0,
    "App::Nopaste" => 0,
    "App::Rad" => 0,
    "App::Software::License" => 0,
    "App::cpanminus" => 0,
    "App::cpanoutdated" => 0,
    "Archive::Tar::Wrapper" => 0,
    "Bundle::CPANxxl" => 0,
    "Bundle::LWP" => 0,
    "Bundle::libnet" => 0,
    "CPAN::Mini" => 0,
    "CPAN::Mini::Devel" => 0,
    "CPAN::Reporter" => 0,
    "CPAN::Uploader" => 0,
    "Cache::Cache" => 0,
    "Cache::FileCache" => 0,
    "DBD::Pg" => 0,
    "DBD::SQLite" => 0,
    "DBD::mysql" => 0,
    "DBI" => 0,
    "DBIx::Class" => 0,
    "DBIx::Class::Schema::Loader" => 0,
    "DBIx::Class::Schema::Versioned" => 0,
    "DBIx::Class::TimeStamp" => 0,
    "Data::DPath" => 0,
    "Data::Visitor" => 0,
    "Data::YAML" => 0,
    "DateTime" => 0,
    "DateTime::Format::Builder" => 0,
    "DateTime::Format::Mail" => 0,
    "DateTime::Format::MySQL" => 0,
    "DateTime::Format::Natural" => 0,
    "DateTime::Format::Strptime" => 0,
    "Devel::Backtrace" => 0,
    "Devel::Cover" => 0,
    "Devel::NYTProf" => 0,
    "Devel::bt" => 0,
    "Digest::SHA1" => 0,
    "Directory::Scratch" => 0,
    "Dist::Zilla" => 0,
    "Dist::Zilla::PluginBundle::SCHWIGON" => 0,
    "Dist::Zilla::PluginBundle::TAPPER" => 0,
    "File::ShareDir" => 0,
    "File::Slurp" => 0,
    "File::Type" => 0,
    "Hash::Merge" => 0,
    "Hash::Merge::Simple" => 0,
    "JSON" => 0,
    "JSON::Syck" => 0,
    "JSON::XS" => 0,
    "List::AllUtils" => 0,
    "List::MoreUtils" => 0,
    "Log::Log4perl" => 0,
    "Module::Install" => "0.95",
    "Module::Install::AuthorRequires" => 0,
    "Module::Install::AuthorTests" => 0,
    "Module::Install::CheckConflicts" => 0,
    "Module::Install::ExtraTests" => 0,
    "Moose" => 0,
    "Moose::Autobox" => 0,
    "MooseX::Daemonize" => 0,
    "MooseX::Declare" => 0,
    "MooseX::LazyRequire" => 0,
    "MooseX::Log::Log4perl" => 0,
    "MooseX::Method::Signatures" => 0,
    "MooseX::Types::Common" => 0,
    "MooseX::Types::DateTime" => 0,
    "MooseX::Types::Email" => 0,
    "MooseX::Types::LoadableClass" => 0,
    "MooseX::Types::Path::Class" => 0,
    "MooseX::Types::URI" => 0,
    "Net::Daemon" => 0,
    "Net::SSH" => 0,
    "Net::SSH::Expect" => 0,
    "Net::Server" => 0,
    "Net::Server::PreForkSimple" => 0,
    "Net::TFTP" => 0,
    "PadWalker" => 0,
    "Parse::RecDescent" => 0,
    "Perl::Tidy" => 0,
    "Perl::Version" => "1.010",
    "Pod::Coverage::TrustPod" => 0,
    "Pod::Weaver::PluginBundle::SCHWIGON" => 0,
    "Pod::Weaver::PluginBundle::TAPPER" => 0,
    "Regexp::Common" => 0,
    "SQL::Translator" => 0,
    "Sub::Exporter" => 0,
    "TAP::DOM" => 0,
    "TAP::Formatter::HTML" => 0,
    "TAP::Parser::Aggregator" => 0,
    "Task::PerlFormance" => 0,
    "Task::SDK::Moose" => 0,
    "Template" => 0,
    "Term::ReadLine::Gnu" => 0,
    "Term::ReadLine::Perl" => 0,
    "Test::Aggregate" => 0,
    "Test::Deep" => 0,
    "Test::Fixture::DBIC::Schema" => 0,
    "Test::MockModule" => 0,
    "Test::Pod" => 0,
    "Test::Pod::Coverage" => 0,
    "Test::Reporter" => 0,
    "Test::Reporter::Transport::Socket" => 0,
    "Test::WWW::Mechanize::Catalyst" => 0,
    "TryCatch" => 0,
    "URI" => 0,
    "URI::Escape" => 0,
    "V" => 0,
    "YAML" => 0,
    "YAML::Syck" => 0,
    "YAML::XS" => 0,
    "bareword::filehandles" => 0,
    "indirect" => 0,
    "multidimensional" => 0,
    "strict" => 0,
    "warnings" => 0
  },
  "TEST_REQUIRES" => {
    "Test::More" => 0
  },
  "VERSION" => "0.003",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  my $tr = delete $WriteMakefileArgs{TEST_REQUIRES};
  my $br = $WriteMakefileArgs{BUILD_REQUIRES};
  for my $mod ( keys %$tr ) {
    if ( exists $br->{$mod} ) {
      $br->{$mod} = $tr->{$mod} if $tr->{$mod} > $br->{$mod};
    }
    else {
      $br->{$mod} = $tr->{$mod};
    }
  }
}

unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



